packages/devops/scripts/deployment/restoreE2eDataFromS3.sh

```diff
@@ -1,12 +1,29 @@
 #!/bin/bash
 
+function restore_table() {
+  local table="$1"
+
+  echo "Restoring $table"
+  psql -U tupaia tupaia -c "\copy $table from 'dumps/$table.csv' delimiter ',' csv header"
+}
+
+s3_folder="dumps/e2e"
+
 echo "Restoring E2E data from S3"
 
-echo "Pulling dumps from S3"
-aws s3 cp s3://tupaia/dumps/e2e/dumps.tgz .
+echo "Pulling latest dump from S3"
+s3_dump_path=$(
+  aws s3api list-objects-v2 \
+    --bucket tupaia \
+    --prefix $s3_folder/ \
+    --query 'reverse(sort_by(Contents[?ends_with(Key, `.tgz`)], &LastModified))[:1].Key' \
```

Comment:
Hopefully this query works better than the UI, which only sorts the first page of data rather than all

Reply:
Haha really?! That's unexpected 😆

Reply:
So unexpected I feel it's worth double checking this query option doesn't do the same...

Reply:
I've just checked @kael89 and I do think this will break if we end up with more than one page of dumps...

"Client-side filtering is supported by the AWS CLI client using the --query parameter. This parameter has capabilities the server-side filtering may not have." ~ https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-filter.html

Perhaps the best option is to just disable pagination, and hope that we don't blow things up:
"You can disable pagination by providing the --no-paginate argument" ~ https://docs.aws.amazon.com/cli/latest/reference/s3api/list-objects-v2.html

Reply:
Thanks for checking! "Special capabilities"... ahaha